Readme.md

```diff
@@ -1,95 +1,97 @@
+the official [SassC](http://github.com/sass/sassc) CLI wrapper. It is *guaranteed* to compile
+with the latest code in LibSass master, since it is also used in the CI process. There is no
+limitation here, as you may use any other LibSass implementer to test your LibSass branch!
 
-Well, luckily, [SassC](http://github.com/sass/sassc) is the official binary wrapper for
-LibSass and is *always* kept in sync. SassC is used by continous integration systems in
-LibSass repository. When developing LibSass, it is best to actually
-checkout SassC and develop in that directory with the SassC spec
-and tests there.
-
-We even run Travis tests for SassC!
-
-Tests
+Testing
 -------
 
-Since LibSass is a pure library, tests are run through the [SassSpec](https://github.com/sass/sass-spec) project using the [SassC](http://github.com/sass/sassc) driver.
+Since LibSass is a pure library, tests are run through the [Sass-Spec](https://github.com/sass/sass-spec)
+project using the [SassC](http://github.com/sass/sassc) CLI wrapper. To run the tests against LibSass while
+developing, you can run `./script/spec`. This will clone SassC and Sass-Spec under the project folder and
+then run the Sass-Spec test suite. You may want to update the clones to ensure you have the latest version.
+Note that the scripts in the `./script` folder are mainly intended for our CI needs.
 
-To run tests against LibSass while developing, you can run `./script/spec`. This will clone SassC and Sass-Spec under the project folder and then run the Sass-Spec test suite. You may want to update the clones to ensure you have the latest version.
+Building
+--------
 
-### DEBUG builds
+To build LibSass you need GCC 4.6+ or Clang/LLVM. If your OS is older, you may need to upgrade
+them first (or install clang as an alternative). On Windows, you need MinGW with GCC 4.6+ or VS 2013
+Update 4+. It is also possible to build LibSass with Clang/LLVM on Windows with various build chains
+and/or command line interpreters.
 
-Set the environment variable `DEBUG` to `1` to enable debug builds that can be debugged
-with `gdb`, `lldb` and others. E.g.: use `$ DEBUG=1 ./script/spec` to run the tests with
-a debug build.
+See the [build docs for further instructions](docs/build.md)!
 
-Library Usage
+Compatibility
 -------------
 
-While LibSass is a library primarily written in C++, it provides a simple
-C interface which should be used by most implementers. LibSass does not do
-much on its own without an implementer. This can be a command line tool, like
-[sassc](https://github.com/sass/sassc) or a [binding](docs/implementations.md)
-to your favorite programing language.
-
-If you want to build or interface with LibSass, we recommend to check out the
-documentation pages about [building LibSass](docs/build.md) and
-the [C-API documentation](docs/api-doc.md).
+Current LibSass 3.4 should be compatible with Sass 3.4. Please refer to the [sass compatibility
```

Comment:
The 3.4 branch isn't an official thing. It was an implementation detail because I wasn't ready to drop some large changes into Node Sass. Node Sass now has the latest 3.5 betas on master.